.github/workflows/go.yaml

```diff
@@ -0,0 +1,74 @@
+name: Go
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["**"]
+
+jobs:
+  check:
```

Comment:
I'd recommend we add `Exhaustive` and `Deadcode` too. `Exhaustive` check if the code verifies every possible way on if and switch statements. `Deadcode` checks for code that is never used.

https://github.com/stellar/stellar-disbursement-platform-backend/blob/develop/.github/workflows/ci.yml#L36-L56

Reply:
Good idea! Will do. 👍